lib/govuk_tech_docs.rb

```diff
@@ -70,6 +70,10 @@ def current_page_review
       def format_date(date)
         date.strftime('%-e %B %Y')
       end
+
+      def active_page(page_path)
+        ("/" + current_page.path) == page_path || current_page.data.parent == page_path
```

Comment:
I think this might not work for the homepage, because `page_path` is `/` and `current_page.path` is `index.html`.